schema_salad/tests/test_cg.py

```diff
@@ -20,8 +20,8 @@ def test_load(self):
                 "type": "string"
             }]
         }
-        rs = cg_metaschema.RecordSchema(doc, "http://example.com/", cg_metaschema.LoadingOptions())
-        self.assertEqual("record", rs.type)
+        rs = cg_metaschema.RecordSchema.fromDoc(doc, "http://example.com/", cg_metaschema.LoadingOptions())
+        assert "record" == rs.type
```

Comment:
Why it uses `assert` rather than `self.assertEqual`?

Reply:
it is a feature of `pytest` for cleaner test cases: https://docs.pytest.org/en/latest/assert.html

Reply:
Currently most of the tests uses `unittest` module, which uses `self.assertEqual` and `self.assertRaises` and others.
IMO combining several testing frameworks gets us confusing when we read the code and also when we try to write new tests.
It is better to use a single testing framework.

Reply:
I agree!

We are in fact [only using `py.test` to run the tests](https://github.com/common-workflow-language/schema_salad/blob/master/setup.py#L55) which [supports the older `unittest` syntax](https://docs.pytest.org/en/latest/unittest.html) as well as the nicer `assert` style. [`cwltool` has already been converted over](https://github.com/common-workflow-language/cwltool/pull/895) and I've been converting pieces of `schema_salad` as I touch them.

Reply:
In that case it is better to separate PR.
One is for codegen and another is to completely switch from `unittest` to `pytest` as common-workflow-language/cwltool#895 do.

Reply:
Fixed in https://github.com/common-workflow-language/schema_salad/pull/268